### PR TITLE
Add sign, verify and read key models.

### PR DESCRIPTION
**Pull request by LensArt**
We use some other parts of vault so we added the model to be able to use these. For reference: https://www.vaultproject.io/api/secret/transit/index.html

### CODE REVIEW

**Comment by grounded042** [comment]
Thanks for the PR @LensArt. I've got a couple of comments / questions, but this is looking real nice.

File: src/Vault/Models/Secret/Transit/ReadKeyResponse.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        [JsonProperty("supports_decryption")]
+        public bool? SupportsDecryption { get; set; }
+
+        [JsonProperty("supports_derivation")]
+        public bool? SupportsDerivation { get; set; }
+
+        [JsonProperty("supports_signing")]
+        public bool? SupportsSigning { get; set; }
+    }
+}
```
Comment (grounded042) [inline src/Vault/Models/Secret/Transit/ReadKeyResponse.cs]: Are you missing the `latest_version`, `backup_info`, and `restore_info` fields?

File: src/Vault/Models/Secret/Transit/SignRequest.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        [JsonProperty("input")]
+        public string Input { get; set; }
+
+        [JsonProperty("context")]
+        public string Context { get; set; }
+
+        [JsonProperty("prehashed")]
+        public bool? Prehashed { get; set; }
+    }
+}
```
Comment (grounded042) [inline src/Vault/Models/Secret/Transit/SignRequest.cs]: Are you missing the `key_version` field?

File: src/Vault/Models/Secret/Transit/SignResponse.cs [hunk 3]
```diff
@@ -0,0 +1,10 @@
+﻿using Newtonsoft.Json;
+
+namespace Vault.Models.Secret.Transit
+{
+    public class SignResponse
+    {
+        [JsonProperty("signature")]
+        public string Signature { get; set; }
+    }
+}
```
Comment (grounded042) [inline src/Vault/Models/Secret/Transit/SignResponse.cs]: I think this could also have a `public_key` field.

File: src/Vault/Vault.csproj [hunk 4]
```diff
@@ -7,7 +7,7 @@
     <PackageProjectUrl>https://github.com/Chatham/Vault.NET/</PackageProjectUrl>
     <PackageLicenseUrl>https://github.com/Chatham/Vault.NET/blob/master/LICENSE</PackageLicenseUrl>
     <PackageIconUrl>https://github.com/Chatham/Vault.NET/blob/master/vault.png?raw=true</PackageIconUrl>
-    <PackageVersion>0.9.1.0</PackageVersion>
+    <PackageVersion>0.9.2.0</PackageVersion>
```
Comment (grounded042) [inline src/Vault/Vault.csproj]: We mirror the Vault version and only create new point releases for PRs and such that are not lined up with a newer version of Vault. Could you move this to `0.9.1.1`?

**Review by grounded042 — CHANGES_REQUESTED** [review]
Thanks again for the PR. I have a couple of things mentioned and then we should be good to merge.

**Comment by LensArt** [comment]
Thanks for the feedback, I first based the models on the response examples in the API documentation. I have now updated the models according to your suggestions and looking into the Vault code. This should cover all possible properties.

**Review by grounded042 — APPROVED** [review]

**Comment by grounded042** [comment]
Looks good. Thanks for the contribution @LensArt!

**Pull request merged**